bigtable/benchmarks/scan_throughput_benchmark.cc

```diff
@@ -0,0 +1,154 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <absl/time/time.h>
+#include <chrono>
+#include <future>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include "bigtable/benchmarks/benchmark.h"
+
+/**
+ * @file
+ *
+ * Measure the throughput of `bigtable::Table::ReadRows()`.
+ *
+ * This benchmark measures the throughput of `bigtable::Table::ReadRows()` on a
+ * "typical" table used for serving data.  The benchmark:
+ * - Creates a table with 10,000,000 rows, each row with a single column family,
+ *   but with 10 columns.
```

Comment:
The first bullet point here is repeated in the 2nd and 3rd.

Reply:
Fixed.